libraries/Microsoft.Bot.Builder.Dialogs/Prompts/OAuthPrompt.cs

```diff
@@ -496,15 +496,23 @@ private async Task<PromptRecognizerResult<TokenResponse>> RecognizeTokenAsync(IT
                 }
                 else
                 {
-                    var tokenExchangeResponse = await adapter.ExchangeTokenAsync(
-                        turnContext,
-                        _settings.ConnectionName,
-                        turnContext.Activity.From.Id,
-                        new TokenExchangeRequest()
-                        {
-                            Token = tokenExchangeRequest.Token,
-                        },
-                        cancellationToken).ConfigureAwait(false);
+                    TokenResponse tokenExchangeResponse = null;
+                    try
+                    {
+                        tokenExchangeResponse = await adapter.ExchangeTokenAsync(
+                           turnContext,
+                           _settings.ConnectionName,
+                           turnContext.Activity.From.Id,
+                           new TokenExchangeRequest()
+                           {
+                               Token = tokenExchangeRequest.Token,
+                           },
+                           cancellationToken).ConfigureAwait(false);
+                    }
+                    catch
+                    {
+                        // Ignore Exceptions

```

Comment:
This makes me nervous, why are we ignoring exceptions?

Reply:
see my other comments about fallback in case token exchange fails